drivers/arm/cci/cci.c

```diff
@@ -21,6 +21,7 @@
 #define CCI400_PART_NUM		0x420
 #define CCI500_PART_NUM		0x422
 #define CCI550_PART_NUM		0x423
+#define MT8183_CCI_PART_NUM	0xB00
```

Comment:
make 0xB00 aligned with 0x423

Reply:
This is aligned, it looks like it isn't because of the GitHub UI. However the changes to this driver should be in a different patch.